qtpy/QtWidgets.py

```diff
@@ -8,15 +8,31 @@
 
 """Provides widget classes and functions."""
 
-from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6
+from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, QtModuleNotInstalledError
+from .utils import _getattr_missing_optional_dep
+
+
+_missing_optional_names = {}
+
```

Comment:
```suggestion

def __getattr__(name):
    """Custom getattr to chain and wrap errors due to missing optional deps."""
    raise getattr_missing_optional_dep(
        name, module_name=__name__, optional_names=_missing_optional_names)

```

Moved from the bottom.